tests/phpBackend/regressions/11135.phpt

```diff
@@ -24,4 +24,5 @@ $errcontext)
 }
 
 ?>
---EXPECT--
+--EXPECTREGEX--
```

Comment:
Seems the newer versions of PHPUnit don't allow for empty expectations - it prints `PHPUnit\Runner\Exception: No PHPT expectation found`. So instead we use a regex stating that it should be empty.